generators/app/templates/screen/index.tsx

```diff
@@ -47,6 +48,24 @@ const <%= componentName %>Screen: React.FC<<%= componentName %>ScreenProps> = ({
     },
   })
 
+  const handleFetch = async (): Promise<void> => {
+    log.i(LOG_TAG, 'onHandleFetch')
+    setState(States.loading)
+
+    try {
+      setState(States.default)
+    } catch (error) {
+      const errorState = formatNetworkErrors.isNetworkError(error)
```

Comment:
for new services from `servicesTon` we use:
```suggestion
      const errorState = err instanceof NetworkError
```